kyo-core/shared/src/main/scala/kyo/System.scala

```diff
@@ -0,0 +1,140 @@
+package kyo
+
+import System.Parser
+import java.lang.System as JSystem
+import kyo.kernel.Reducible
+
+abstract class System:
+    def env[E, A](name: String)(using Parser[E, A], Frame): Maybe[A] < (Abort[E] & IO)
+    def property[E, A](name: String)(using Parser[E, A], Frame): Maybe[A] < (Abort[E] & IO)
+    def lineSeparator(using Frame): String < IO
+    def userName(using Frame): String < IO
+    def userHome(using Frame): String < IO
+
+end System
+
+object System:
+
+    val local = Local.init(live)
+
+    val live: System =
```

Comment:
Should this be private?

Reply:
It follows the pattern of `Clock`, `Console`, and `Timer`. I think having it public can use useful. For example, it allows implementing a `System` wrapper class with custom behaviors that dispatches to the live instance.